docs/source/algorithms/EstimateResolutionDiffraction-v1.rst

```diff
@@ -30,11 +30,12 @@ and thus
 
 where,
 
--  :math:`\Delta T` is the time resolution from moderator
--  :math:`\Delta\theta` is the coverage of the detector, and can be
+-  :math:`\Delta T` is the time resolution, generally from moderator dimensions. If supplied through ``DeltaTOF`` parameter, the center wavelength is used to calculate the center time-of-flight. Alternatively, one can specify ``DeltaTOFOverTOF``.
```

Comment:
:math:`\Delta T` is the time resolution, or pulse width, generally derived from the moderator dimensions.